packages/core/src/agent/MessageSender.ts

```diff
@@ -341,49 +332,6 @@ export class MessageSender {
     throw new AriesFrameworkError(`Unable to send message to service: ${service.serviceEndpoint}`)
   }
 
-  private async retrieveServicesFromDid(did: string) {
```

Comment:
Ideally I would like to keep the did module separated as much as possible from didcomm. Maybe we can look at adding a didcomm directory and a didcomm did document service that handles this?

Reply:
I have tried to split it into a `packages/core/src/modules/didcomm/services/DidCommDocumentService`. Have a look if that's what you mean

Reply:
Yes, this is great. We can now move more items from the agent directory to the didcomm module over time.